### DISCUSSION TITLE
Connect to pg database and delete some stuff from the shell

### DISCUSSION THREAD

**Discussion by nelsonmandeladev** [cookiecutter/cookiecutter-django#4088] [Q&A] [answered]
Hi there, I want to connect to the pg shell and delete some data entries manually. If there is a solution using pgAdmin it can also help. I'm running locally using docker.
Thanks

**Comment by browniebroke** [comment, marked as answer]
Assuming the postgres container is alreasdy running, you can open a bash shell prompt with:

```
docker-compose -f local.yml exec postgres bash
```

From there, you can type [psql](https://www.postgresql.org/docs/current/app-psql.html) to run a SQL REPL and execute whatever queries you need. Make sure to connect with the right credentials, they are located in `.envs/.local/.postgres`, which should be in the container environment:

```
psql -d ${POSTGRES_DB} -U ${POSTGRES_USER}
```

If you want to connect with a more visual tool from outside the container, you'll need to map the Postgres port (5432) to your host machine. This is usually done by adding them to the docker compose config file `local.yml`:

```diff
   postgres:
+    ports:
+      - "5432:5432"
     build:
```

More explanations on port mapping [in the docs](https://docs.docker.com/compose/networking/).
